sycl/test-e2e/bindless_images/sampled_fetch/fetch_1D_USM.cpp

```diff
@@ -0,0 +1,104 @@
+// REQUIRES: linux
+// REQUIRES: cuda
```

Comment:
Aspect requirements should be stated here so the test is marked as skipped:
```suggestion
// REQUIRES: cuda
// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_1d_usm
```
Same goes for the other tests.